src/openapi-generator.ts

```diff
@@ -92,6 +93,10 @@ export class OpenAPIGenerator {
       ...config,
       components: this.buildComponents(),
       paths: this.pathRefs,
+      // As this is invalid in Open API 3.0 we optionally set it
```

Comment:
Good catch

Reply:
Hmm instead of this I see that openapi3-ts has a newer major version - 3.1.1. And looking at [this line](https://github.com/metadevpro/openapi3-ts/blob/master/src/model/OpenApi.ts#L32) I think just upgrading the library would be a better shout.

Reply:
Update I realized what the meaning of the optional setting is here. I also saw that you introduced the change there :smile:  I think it would be a good addition for you to update the dependency here as well :smile:

Reply:
Yeah I'm having a few existing type issues upgrading the library. Might raise a PR to separately to update it.